### PR TITLE
lightning: update descriptions of some parameters

### PR DESCRIPTION
**Pull request by hfxsd**
### What is changed, added or deleted? (Required)
as tittle

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v7.0 (TiDB 7.0 versions)
- [x] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-lightning/tidb-lightning-configuration.md [hunk 1]
```diff
@@ -114,8 +114,8 @@ driver = "file"
 # "local"：Physical Import Mode，默认使用。适用于 TB 级以上大数据量，但导入期间下游 TiDB 无法对外提供服务。
 # "tidb"：Logical Import Mode。TB 级以下数据量可以采用，下游 TiDB 可正常提供服务。
 # backend = "local"
-# 是否允许向已存在数据的表导入数据。默认值为 false。
-# 当使用并行导入模式时，由于多个 TiDB Lightning 实例同时导入一张表，因此此开关必须设置为 true。
+# 是否允许启动多个 TiDB Lightning 实例并行导入到一个或多个目标表。默认取值为 false。
```
Comment (Frank945946) [inline tidb-lightning/tidb-lightning-configuration.md:117]:
```suggestion
# 是否允许启动多个 TiDB Lightning 实例（physical 导入模式）并行导入到一个或多个目标表。默认取值为 false。
```

Comment (Frank945946) [reply tidb-lightning/tidb-lightning-configuration.md:117]: @gozssky  帮忙看一下是否仅限 local backend

File: tidb-lightning/tidb-lightning-configuration.md [hunk 2]
```diff
@@ -114,8 +114,8 @@ driver = "file"
...
+# 多个 TiDB Lightning 实例同时导入一张表时，此开关必须设置为 true。
```
Comment (Frank945946) [inline tidb-lightning/tidb-lightning-configuration.md:118]:
```suggestion
# 多个 TiDB Lightning 实例（physical 导入模式）同时导入一张表时，此开关必须设置为 true。但前提是目标表不能存在数据，即所有的数据都只能是由 Lightning 导入。
```

Comment (Frank945946) [reply tidb-lightning/tidb-lightning-configuration.md:118]: @gozssky 帮忙 Review 一下

File: tidb-lightning/tidb-lightning-configuration.md (hunk 1 above)
Comment (gozssky) [reply tidb-lightning/tidb-lightning-configuration.md:117]: 是的。要不要添加一个注意事项，incremental-import 不意味着支持增量导入。

**Review by Frank945946 — APPROVED** [review]

**Review by gozssky — APPROVED** [review]

File: tidb-lightning/tidb-lightning-configuration.md [hunk 3]
```diff
@@ -114,8 +114,8 @@ driver = "file"
 # "local"：Physical Import Mode，默认使用。适用于 TB 级以上大数据量，但导入期间下游 TiDB 无法对外提供服务。
 # "tidb"：Logical Import Mode。TB 级以下数据量可以采用，下游 TiDB 可正常提供服务。
 # backend = "local"
-# 是否允许向已存在数据的表导入数据。默认值为 false。
-# 当使用并行导入模式时，由于多个 TiDB Lightning 实例同时导入一张表，因此此开关必须设置为 true。
+# 是否允许启动多个 TiDB Lightning 实例（**physical import mode**）并行导入到一个或多个目标表。默认取值为 false。注意，这个参数**不是用于增量导入数据**，仅限目标表为空的场景使用。
+# 多个 TiDB Lightning 实例（physical import mode）同时导入一张表时，此开关必须设置为 true。但前提是目标表不能存在数据，即所有的数据都只能是由 Lightning 导入。
```
Comment (hfxsd) [inline tidb-lightning/tidb-lightning-configuration.md:118]:
```suggestion
# 多个 TiDB Lightning 实例（physical import mode）同时导入一张表时，此开关必须设置为 true。但前提是目标表不能存在数据，即所有的数据都只能是由 TiDB Lightning 导入。
```

**Review by ran-huang — APPROVED** [review]

**Pull request merged**